egg-herbie/src/math.rs

```diff
@@ -115,7 +115,7 @@ impl Default for ConstantFold {
 
 impl Analysis<Math> for ConstantFold {
     type Data = Option<(Constant, (PatternAst<Math>, Subst))>;
-    fn make(egraph: &EGraph, enode: &Math) -> Self::Data {
+    fn make(egraph: &mut EGraph, enode: &Math) -> Self::Data {
```

Comment:
What does this do? Isn't it bad? API change?

Reply:
Breaking change on the egg side. I'm not entirely sure why it was made